apparmor.d/profiles-m-r/mutt

```diff
@@ -0,0 +1,178 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2023 Zane Zakraisek <zz@eng.utah.edu>
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/3.0>,
+
+include <tunables/global>
+
+@{exec_path} = /{usr/,}bin/mutt
+profile mutt @{exec_path} {
+  include <abstractions/base>
+  include <abstractions/nameservice>
+  include <abstractions/openssl>
+  include <abstractions/consoles>
+
+  @{exec_path} mr,
+
+  # Mutt config files
+  /usr/{local/,}share/mutt/Muttrc r,
```

Comment:
This should be simplied as:
```
/usr/share/mutt/** r,
```

Therefore other instances of `/usr/share/mutt/` can be removed. Also local files are usually out of scope of apparmor profile (that's not a project rule, but an apparmor one, you will never confine something in `/usr/local/foo` because you don't know how these program have been modified from upstream)

Reply:
ah nice catch